app/bundles/LeadBundle/Form/DataTransformer/FieldFilterTransformer.php

```diff
@@ -55,7 +55,7 @@ public function transform($rawFilters)
                 $rawFilters[$k] = array_merge($this->default, $rawFilters[$k]);
             }
             if ('datetime' == $f['type']) {
-                if (in_array($f['filter'], $this->relativeDateStrings) or stristr($f['filter'][0], '-') or stristr($f['filter'][0], '+')) {
+                if (!$f['filter'] || in_array($f['filter'], $this->relativeDateStrings) or stristr($f['filter'][0], '-') or stristr($f['filter'][0], '+')) {
```

Comment:
```suggestion
                if (!$f['filter'] || in_array($f['filter'], $this->relativeDateStrings) || stristr($f['filter'][0], '-') || stristr($f['filter'][0], '+')) {
```

`||` vs `or`  
Not affected by the PR but the mixed style within the condition hurts my eye and Mautic follows symfony conding style, so this should be `||` everywhere.

Reply:
Yes, you're true. I forgot change old code. Thanks